Index: validation-templates/src/main/kotlin/com/validation/typeclass/DeferValidator.kt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- validation-templates/src/main/kotlin/com/validation/typeclass/DeferValidator.kt	(date 1584671832159)
+++ validation-templates/src/main/kotlin/com/validation/typeclass/DeferValidator.kt	(date 1584671832159)
@@ -0,0 +1,8 @@
+/* gakshintala created on 3/14/20 */
+package com.validation.typeclass
+
+import arrow.fx.typeclasses.MonadDefer
+
+interface DeferValidator<F, S, E>: MonadDefer<F> {
+    val validatorAE: ValidatorAE<S, E>
+}
